cves/kernel/CVE-2016-5728.yml

```diff
@@ -448,7 +475,10 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: | 
```

Comment:
The first sentence feels like a bullet point. I would suggest changing this to flow like a paragraph all the way through.